README.md

```diff
@@ -1,38 +1,128 @@
 # BentoMap
+> Map Clustering for Swift
 
 [![Build Status](https://travis-ci.org/Raizlabs/BentoMap.svg?branch=develop)](https://travis-ci.org/Raizlabs/BentoMap)
 [![Version](https://img.shields.io/cocoapods/v/BentoMap.svg?style=flat)](http://cocoapods.org/pods/BentoMap)
 [![License](https://img.shields.io/cocoapods/l/BentoMap.svg?style=flat)](http://cocoapods.org/pods/BentoMap)
 [![Platform](https://img.shields.io/cocoapods/p/BentoMap.svg?style=flat)](http://cocoapods.org/pods/BentoMap)
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 
-BentoMap is an implementation of [quadtrees](https://en.wikipedia.org/wiki/Quadtree) for map annotation clustering and storage written in Swift. The library intends to require minimal code in-app to get clustered annotations on screen, the example target included in BentoMap.xcodeproj contains the bare code required dynamically update the map with clusters whenever the map view's bounds changes.
+BentoMap is an implementation of [quadtrees](https://en.wikipedia.org/wiki/Quadtree) originally implemented for map annotation clustering and storage written in Swift.
 
-The Android equivalent, known as "Marker Clustering," is [documented here](https://developers.google.com/maps/documentation/android-api/utility/marker-clustering).
+This includes a data structure with generics to store data used to generate annotations, the ability to query which annotations are included in a region with a bucket size used to generate clusters, and protocols that can be adopted by other data types to allow storage and retrieval from `BentoBox` containers.
 
-## Installation with CocoaPods
+For more information, check out the [Raizlabs Developer Blog][rl]. The Android equivalent, known as "Marker Clustering," is [documented here][mk].
 
-BentoMap is available through [CocoaPods](http://cocoapods.org). To install
-it, simply add the following line to your Podfile:
+![BentoMap](Resources/bento_animation3.gif)
+
+## Features
+
+- [x] Store annotation data in QuadTrees
+- [x] Fetch annotations in a region, with a clustering threshold
+- [x] Protocols for storing other data types
+
+## Requirements
+
+- iOS 9.0+
+- Xcode 8.0
+
+## Installation
+
+#### CocoaPods
+You can use [CocoaPods](http://cocoapods.org/) to install `BentoMap` by adding it to your `Podfile`:
 
 ```ruby
+platform :ios, '9.0'
+use_frameworks!
 pod 'BentoMap'
```

Comment:
#? wouldn't the `pod` portion typically be inside a target specifier? It's pretty typical to just put the single line for your pod. It's understood that it's within the larger context of a Podfile. Or did you have specific requests to clarify it?